Solving Issue #4071 (#4241)
MicrosoftDocs/windows-itpro-docs: windows/security/identity-protection/hello-for-business/hello-overview.md

## The difference between Windows Hello and Windows Hello for Business

> - Individuals can create a PIN or biometric gesture on their personal devices for convenient sign-in. This use of Windows Hello is unique to the device on which it is set up, using a simple password hash, however it is not backed by asymmetric (public/private key) or certificate-based authentication. 

Reviewer: ```suggestion
- Individuals can create a PIN or biometric gesture on their personal devices for convenient sign-in. This use of Windows Hello for Business is unique to the device on which it is set up, using a simple password hash. However, it is not backed by asymmetric (public/private key) or certificate-based authentication. 
```

> - Individuals can create a PIN or biometric gesture on their personal devices for convenient sign-in. This use of Windows Hello is unique to the device on which it is set up, using a simple password hash. However, it is not backed by asymmetric (public/private key) or certificate-based authentication. 

Reviewer 2: This original statement isn't accurate. An individual can create a Windows Hello container with MSA that leverages asymmetric key authentication. It is also misleading because the title and statement implies that using the convenience mechanism is the only way to use Windows Hello vs Windows Hello for business

Author: Once again if you have documentation, share it. I'm not a "windows hello" or "windows hello for business expert"

Reviewer 2: I think you should modify the statement to: 
"Individuals can create a PIN or biometric gesture on their personal devices for convenient sign-in. This use of Windows Hello is unique to the device on which it is set up, but can use a simple password hash depending on an individual's account type. This configuration is referred to as Windows Hello convenience PIN and it is not backed by asymmetric (public/private key) or certificate-based authentication."

- **Windows Hello for Business**, which is configured by Group Policy or mobile device management (MDM) policy, uses key-based or certificate-based authentication. This makes it much more secure than **Windows Hello**.

> 

Reviewer 2: This statement also implies that Windows Hello is always less secure than Windows Hello for Business, which isn't always the case. It may be better to rewrite this section to specify that Windows Hello can be deployed as a convenience mechanism or can be deployed leveraging asymmetric keys/certs but that Windows Hello for Business **ONLY** only ever refers to when keys/certs are deployed.

Author: @mapalko If you have a reference that I could use to add value. Please let me know, personally I haven't work on this "hello vs hello business" distinction if you want or feel that it should be addressed differently, please let me know. I have researched a bit and point what I thought it was useful to solve the issue already.

Reviewer 2: @j0rt3g4 unfortunately these nuances aren't well documented which is why this problem exists in the first place. My main issue is with the original copy. Your changes are a definite improvement over it.

My suggestion is to to add "always" to the first sentence around being backed by key and change "Windows Hello" to "Windows Hello convenience PIN"
**Windows Hello for Business**, which is configured by Group Policy or mobile device management (MDM) policy, always uses key-based or certificate-based authentication. This makes it much more secure than **Windows Hello convenience PIN**.

Author: Alright @mapalko this is added.

> - Windows Hello for Business, which is configured by Group Policy or mobile device management (MDM) policy, uses key-based or certificate-based authentication which increases significatively the security respect the regular Windows Hello.

Reviewer: ```suggestion
- Windows Hello for Business, which is configured by Group Policy or mobile device management (MDM) policy, uses key-based or certificate-based authentication which increases significantly the security in relation to the regular Windows Hello.
```

> - Windows Hello for Business, which is configured by Group Policy or mobile device management (MDM) policy, uses key-based or certificate-based authentication which increases significatively the security respect the regular Windows Hello.

Reviewer 3: ```suggestion
- Windows Hello for Business, which is configured by Group Policy or mobile device management (MDM) policy, uses key-based or certificate-based authentication. This makes it much more secure than Windows Hello.
```

Author: Added in PR 41814c6
